src/facility.ts

```diff
@@ -0,0 +1,72 @@
+import axios from "axios";
+
+var axiosInstance = axios.create({
+    baseURL: process.env.REACT_APP_FACILITY_API_PREFIX
+});
+
+export enum UserActionTypes {
+    GETINFO = "USER/GETACTION",
+    LOADING = "USER/LOADING"
+}
+
+export interface IUser {
+    userid: number,
+    username: string,
+    globus_email: string,
```

Comment:
Why can't this just be `email`? I don't get globus, also I think we should be using `camelCase` consistently on the TypeScript side.

Reply:
If the variables come from the server we should just use what the server uses to be clear, I don't think there is value in rewriting and it is the norm to pass through what server APIs give.

Reply:
the facility API had globus email before, but that has been changed now to email and orcid.

all lowercase for email, username, and orcid